Rollbar/RollbarConfiguration.m

```diff
@@ -118,86 +98,55 @@ - (id)initWithLoadedConfiguration {
     return self;
 }
 
-// Rollbar enabled flag:
-@synthesize enabled = _enabled;
```

Comment:
`synthesize` is almost never required and should not be used if the ivar synthesis is automatic. It was only necessary in the long, long ago.